assets/js/alma-widgets-inject.js

```diff
@@ -0,0 +1,82 @@
+/**
+ * Inject payment plan widget.
+ *
+ * @package Alma_WooCommerce_Gateway
+ */
+
+(function () {
+	var paymentPlanContainerId = "#alma-payment-plan";
+	var paymentPlanContainer   = jQuery( paymentPlanContainerId );
```

Comment:
Perso j'aime bien que les refs jQuery soient précédées d'un `$`
```suggestion
	var $paymentPlanContainer   = jQuery( paymentPlanContainerId );
```